api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -93,6 +98,12 @@ public void executeSQL_shouldExecuteSqlContainingGroupBy() {
 		adminService.executeSQL(sql2, true);
 	}
 	
+	@Test
+	public void executeSQL_shouldFailIfSqlIsNull() {
```

Comment:
Can you create a new class `AdministrationServiceUnitTest` where you put this test. This test class extends `BaseContextSensitiveTest` which makes all these tests rely on an in-memory DB. There are quite a few cases where we dont need the DB, like this test. If you look at the `AdministrationServiceImpl` we just want to make sure the method returns `null` and that the DAO method is never called, meaning the SQL is never executed.

You can use #2536 as a guide on how to setup the mocks.

Reply:
@teleivo I will create a new class and follow your steps. Thanks for your time.